wgpu-hal/src/vulkan/adapter.rs

```diff
@@ -1078,17 +1109,17 @@ impl super::Adapter {
                 lang_version: (1, 0),
                 flags,
                 capabilities: Some(capabilities.iter().cloned().collect()),
-                bounds_check_policies: naga::back::BoundsCheckPolicies {
-                    index: naga::back::BoundsCheckPolicy::Restrict,
+                bounds_check_policies: naga::proc::BoundsCheckPolicies {
+                    index: naga::proc::BoundsCheckPolicy::Restrict,
```

Comment:
ideally, we'd land this in a separate PR, but this isn't blocking

Reply:
Decided to keep this in if it's not blocking, as the multiview builtin update is necessary for testing this functionality for me personally.